Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1199,6 +1199,7 @@ open class NavigationMapView: UIView {
     var routeRemainingDistancesIndex: Int?
     var fractionTraveled: Double = 0.0
     var currentLegIndex: Int?
+    var offRouteDistanceCheckDisabled: Bool = false
```

Comment:
What about using `offRouteDistanceCheckEnabled`? I think it'll be easier to understand, as you're checking it with unary operator anyway.

Reply:
Thanks for the suggestion, I'll change it to `offRouteDistanceCheckEnabled`.